platform/linux-generic/odp_ishm.c

```diff
@@ -2045,7 +2065,7 @@ int _odp_ishm_status(const char *title)
 		_ODP_PRINT("%2i  %-*s %s%c  %p-%p %-8" PRIu64 "   "
 			  "%-8" PRIu64 " %-3" PRIu64 " %-3" PRIu64 " "
 			  "%-3d %s\n",
-			  i, max_name_len, ishm_tbl->block[i].name,
+			  i, max_name_len, ishm_tbl->block[i].param_name,
```

Comment:
Since .name was changed to .param_name. the corresponding field width parameter max_name_len should also be calculated from param_name instead of name.

Reply:
Fixed in V3.